BeVolt/App/src/main.c

```diff
@@ -269,21 +269,121 @@ int main(){
 #include "SPI.h"
 #include "stm32f4xx.h"
 
+void checkModuleTemperatureTest(void);
+void checkIndividualSensorTest(void);
+void checkDangerTest(void);
+void checkMasterTest(void);						// To implement later. Able to check everything with easeee
 
+extern int16_t ModuleTemperatures[NUM_TEMPERATURE_BOARDS][20];
 
 
 int main(){
 	UART3_Init(9600);
```

Comment:
@SijWoo Does printf use UART1 or UART3?

Reply:
For master v1.1, it'll use UART3. Once everyone is done testing on master v1.0 and we have the nucleo PCB, I'm going to fix up UART.c to only contain code for v1.1 and deprecate the v1.0 board